scripts/sysman/frequency.yml

```diff
@@ -146,6 +146,8 @@ etors:
       desc: "Overclock interpolative mode - In this mode, the voltage/frequency curve can be extended with a new voltage/frequency point that will be interpolated. The existing voltage/frequency points can also be offset (up or down) by a fixed voltage. This mode disables FIXED and OVERRIDE modes."
     - name: FIXED
       desc: "Overclocking fixed Mode - In this mode, hardware will disable most frequency throttling and lock the frequency and voltage at the specified overclock values. This mode disables OVERRIDE and INTERPOLATIVE modes. This mode can damage the part, most of the protections are disabled on this mode."  
+details:
+    - "1.6": "Note: This enum is getting  deprecated at 2.0 version."
```

Comment:
could we add information on what the users need to do to get the same functionality in 2.0? or is it that this is expected to not be useful anymore?

Reply:
This particular API's are not useful anymore as it is designed for DG2 and not useful after. Even for DG2, this particular API's are not getting used as the overclocking is handled by a different API (IGCL).  For newer products  the overclocking  is addressed on a dedicated API section 'Overclock'

Reply:
No APIs should be designed for a specific product. Should aim to design L0 such that it is generically useful for any accelerator device.

Reply:
@bmyates  is correct. We should look at APIs in a holistic manner. So following @sanilkumar0 answer, the question would be:

do we think the interfaces are not useful for any current or future device? From above, it seems that the functionality is replaced by the Overclock APIs, correct @sanilkumar0 ?